README.md

```diff
@@ -6,7 +6,7 @@
 - CSS
 
 ## Live Demo
-- Live version: NO LIVE VERSION YET
+- Live version: [mrjuda.github.io/judaport](https://mrjuda.github.io/judaport)
```

Comment:
- [ ] Hi you've made I change here however, I notice it is the only change you've made. please Look at the previous reviewer's comments and make all his suggested improvements. 
- [ ] Have a look at the inline comments and kindly make the necessary changes.

Reply:
done it